Discord.Addons.Interactive/Discord.Addons.Interactive.csproj

```diff
@@ -23,6 +23,10 @@
     
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+  </PropertyGroup>
```

Comment:
is this intentional?

Reply:
It is, so we'll also have debug info in Visual Studio's console.